packages/config/src/projects/other/da-beat/blockchain/memo/memo.ts

```diff
@@ -0,0 +1,69 @@
+import { NO_BRIDGE } from '../../templates/no-bridge-template'
+import { DaEconomicSecurityRisk } from '../../types/DaEconomicSecurityRisk'
+import { DaFraudDetectionRisk } from '../../types/DaFraudDetectionRisk'
+import { DaLayer } from '../../types/DaLayer'
+import { linkByDA } from '../../utils/link-by-da'
+
+export const memo: DaLayer = {
+  id: 'memo',
+  type: 'DaLayer',
+  kind: 'PublicBlockchain',
+  display: {
+    name: 'Meeda',
+    slug: 'memo',
+    description: `Meeda (MemoDA) is a blockchain-based cloud storage protocol developed by MEMO Labs.`,
+    links: {
+      websites: ['https://www.memolabs.org/'],
+      documentation: ['https://memolabs.gitbook.io/meeda'],
+      repositories: ['https://github.com/memoio'],
+      apps: [],
+      explorers: ['https://scan.metamemo.one:8080/'],
+      socialMedia: [
+        'https://x.com/MemoLabsOrg',
+        'https://discord.com/invite/YG4Ydv2E7X',
+      ],
+    },
+  },
+  consensusAlgorithm: {
+    name: '',
+    description: ``,
+    blockTime: 0, // seconds average
+    consensusFinality: 0,
+    unbondingPeriod: 0,
+  },
+  technology: `
+    ## Architecture
+    Meeda's architecture consists of several components. 
+    Storage Nodes store blob data and generate data availability commitments, with node deployment available to all users. 
+    Light Nodes are responsible for challenging storage nodes through on-chain verification, and their deployment is also open to all users. 
+    The Operator, currently deployed by the project itself, handles the erasure coding of data and data chunks distribution, as well as the aggregation of KZG commitment proofs from Storage Nodes. 
+    
+    Storage Nodes, based on seed information generated periodically on the Memo chain, submit KZG data commitments to the Operator. 
+    The Verification Contract is responsible for on-chain verification on Memo chain, and it will generate seed information through the VRF-Sampling (Verifiable Random Function) cycle to determine which files to select for verification.
```

Comment:
```suggestion
    The Verification Contract is responsible for onchain verification on Memo chain, and it will generate seed information through the VRF-Sampling (Verifiable Random Function) cycle to determine which files to select for verification.
```